@LinProp/LinProp.m

```diff
@@ -1,42 +1,58 @@
+%   <a href="matlab:help LinProp.get_idof -displayBanner"               >get_idof</a>                Inverse degrees of freedeom
+%   <a href="matlab:help LinProp.get_moment -displayBanner"             >get_moment</a>              n'th central moment
+%   <a href="matlab:help LinProp.unc_budget -displayBanner"             >unc_budget</a>              Opens budget window (LinProp only)
 %
-% LinProp Input (ComplexUncArray)
-% [a] = LinProp([value], [covariance], (description))
+% <strong>Interpolation and Integration Methods</strong>
+%   <a href="matlab:help LinProp.integrate -displayBanner"          >integrate</a>           Integration with cumulative result
+%   <a href="matlab:help LinProp.integrate2 -displayBanner"         >integrate2</a>          Integration with scalar result
+%   <a href="matlab:help LinProp.interpolation -displayBanner"      >interpolation</a>       Interpolation
+%   <a href="matlab:help LinProp.interpolation2 -displayBanner"     >interpolation2</a>      Interpolation with linear unc. propagation
+%   <a href="matlab:help LinProp.spline -displayBanner"             >spline</a>              Spline interpolation
+%   <a href="matlab:help LinProp.spline2 -displayBanner"            >spline2</a>             Spline interpolation with linear unc. propagation
+%   <a href="matlab:help LinProp.splinecoefs -displayBanner"        >splinecoefs</a>         Coefficients of interpolation spline
+%   <a href="matlab:help LinProp.splineintegrate -displayBanner"    >splineintegrate</a>     Spline integration with cumulative result
+%   <a href="matlab:help LinProp.splineintegrate2 -displayBanner"   >splineintegrate2</a>    Spline integration with scalar result
 %
-% LinProp From Samples
-% a = LinProp([samples], 'samples', (description), (probability))
+% <strong>Object Behavior</strong>
+% Scalar LinProp objects behave like MATLAB fundamental types with respect
+% to copy operations. Copies are independent values. Operations that you
+% perform on one object do not affect copies of that object.
+% Non-scalar LinProp objects are referenced by their handle variable.
+% Copies of the handle variable refer to the same object. Operations that
+% you perform on a handle object are visible from all handle variables that
+% reference that object.
 %
-% LinProp Xml String
-% a = LinProp(xml_string)
-%
-% LinProp Xml File
-% a = LinProp(filepath, 'xml_file')
-%
-% LinProp Binary File
-% a = LinProp(filepath, 'binary_file')
-%
-% LinProp System (RealUncNumber)
-% a = LinProp(value, [sys_inputs], [sys_sensitivities], 'system')
-%
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, idof, id, description)
+% B = <a href="matlab:help LinProp.copy -displayBanner">copy</a>(A) copies each element in the array of handles A to a new
+% array of handles B.
+
+% Metas.UncLib.Matlab.LinProp V2.6.0
+% Michael Wollensack METAS - 06.07.2022
+% Dion Timmermann PTB - 22.06.2022
```

Comment:
What would be the drawback of putting the following lines to the begin of the file:

```
% Metas.UncLib.Matlab.LinProp V2.6.0
% Michael Wollensack METAS - 06.07.2022
% Dion Timmermann PTB - 22.06.2022
```

Reply:
This is only a question of style. Usually, the matlab help texts do not contain a copyright notice. Putting these lines at the begining of the file would cause them to be added to the top of the help texts.

With normal MATLAB commands, the help text to starts with a short summary of the function, as e.g. in 
```
>> help sin
 sin    Sine of argument in radians.
    sin(X) is the sine of the elements of X.
 
    See also asin, sind, sinpi.

    Documentation for sin
    Other functions named sin
```


If we move these lines back to the top, the start of `help LinProp` and `doc LinProp` would look as follows:
![grafik](https://user-images.githubusercontent.com/14871355/201616076-53d8b4e5-1b0b-453e-9a14-a57e3010793f.png)
![grafik](https://user-images.githubusercontent.com/14871355/201616364-c9e66682-eb9f-4902-8753-dae58c77d095.png)

With the help/doc texts being so long, and this being a custom library with version numbers independent of the current matlab version, I see arguments to move these lines back to the top of the file.

Reply:
As I suspect that is what you prefer, I have moved these lines back to the top of the file in [b71ab10](https://github.com/wollmich/metas-unclib-matlab-wrapper/pull/51/commits/b71ab1079922bc573624e3ec3093883027734508) . If you have a different opinion, we can simply revert the changes in the commit.